src/pybind/mgr/ssh/module.py

```diff
@@ -608,6 +608,33 @@ def run(host, host_info):
 
         return SSHReadCompletion(results)
 
+    @log_exceptions
+    def blink_device_light(self, ident_fault, on, locs):
+        def blink(host, dev, ident_fault, on):
+            cmd = [
+                'lsmcli',
+                'local-disk-%s-led-%s' % (
```

Comment:
Better use the new [Python formatting](https://pyformat.info/#simple) here and the rest of the PR.

Reply:
whine: the % style is so much less typing...

Let's do a pass over the whole script in a separate pr that converts it all if it matters?